### PR TITLE
five layer burrito description spelling

### PR DESCRIPTION
**Pull request by Ktlwjec1**
title

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 1]
```diff
@@ -260,7 +260,7 @@
 
 /obj/item/reagent_containers/food/snacks/fivelayerburrito
 	name = "five layer burrito"
-	desc = "5 burritos stacked ontop of eachother, looks suspicously like a layer manifold"
+	desc = "Five burritos stacked on top of each other, looks suspiciously like a layer manifold."
```
Comment (jcatjmeow) [inline code/modules/food_and_drinks/food/snacks_other.dm]:
```suggestion
	desc = "Five burritos stacked on top of each other. It looks suspiciously like a layer manifold."
```

**Review by jcatjmeow — DISMISSED** [review]
grammer

File: code/modules/food_and_drinks/food/snacks_other.dm (hunk 1 above)
Comment (Hopekz) [inline code/modules/food_and_drinks/food/snacks_other.dm]:
```suggestion
	desc = "Five 🌯burritos 🌯 stacked 🔛on🔛🔛 ⬆️top⬆️ of each other. It looks👀 suspiciously like👍👍 a🅰️ layer manifold."
```

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Review by redmoogle — APPROVED** [review]
~~Oops~~ Goob pr 👍

**Pull request merged**